src/build/iconGenerator.js

```diff
@@ -113,9 +163,30 @@ function buildJsonStructure(iconsFolderBasePath) {
  */
 function getDefaultSchema(iconsFolderBasePath) {
   /* eslint-disable no-underscore-dangle */
-  defaultSchema.iconDefinitions._file.iconPath = iconsFolderBasePath + 'file.svg';
-  defaultSchema.iconDefinitions._folder.iconPath = iconsFolderBasePath + 'folder.svg';
-  defaultSchema.iconDefinitions._folder_open.iconPath = iconsFolderBasePath + 'folder_opened.svg';
+
+  // dark theme
+  defaultSchema.iconDefinitions._file
+    .iconPath = iconsFolderBasePath + 'file.svg';
+  defaultSchema.iconDefinitions._folder
+    .iconPath = iconsFolderBasePath + 'folder.svg';
+  defaultSchema.iconDefinitions._folder_open
+    .iconPath = iconsFolderBasePath + 'folder_opened.svg';
+
+  // light theme
+  // default file and folder related icon paths if not set,
```

Comment:
I'm used in using single-line comment style inside functions/methods even if they are multi-lined.
I only use comment-block styling if they are file header (usually license) or above functions/methods as descreptive comments.
If @robertohuertasm wished to follow the comment-block style inside functions I have no problem switching to that.